src/omv/imlib/framebuffer.c

```diff
@@ -170,23 +170,20 @@ void framebuffer_update_jpeg_buffer() {
                 bool overflow = false;
                 if (0) {
                 #if OMV_RAW_PREVIEW_ENABLE
-                } else if (src->is_mutable &&
-                           image_size(src) <= OMV_JPEG_BUFFER_SIZE_MAX) {
```

Comment:
Since this is constant there isn't a need to compare it in run time. I tried adding a macro... but, the since the jpeg size is a linker variable it can't be compared by the pre-processor.

Reply:
I'm not following. So the solution is not to check at all to save 1 branch ?

Reply:
If a board is misconfigured this will save a lot of debugging time.

Reply:
Yeah, will add back. It should be checked.

Reply:
Done